gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -524,7 +587,7 @@ public static Builder builder() {
    * @return a complete blob information.
    * @throws StorageException upon failure
    */
-  BlobInfo create(BlobInfo blobInfo, InputStream content, BlobTargetOption... options);
+  BlobInfo create(BlobInfo blobInfo, InputStream content, BlobWriteOption... options);
```

Comment:
Shall we add a comment in the javadoc (here and in writer)  that "by default any md5 and crc32c values in the given blobInfo are ignored unless requested via the BlobWriteOption.md5Match and BlobWriteOption.crc32cMatch options"?